libbeat/outputs/elasticsearch/config.go

```diff
@@ -15,7 +15,7 @@ type elasticsearchConfig struct {
 	ProxyURL         string             `config:"proxy_url"`
 	LoadBalance      bool               `config:"loadbalance"`
 	CompressionLevel int                `config:"compression_level" validate:"min=0, max=9"`
-	TLS              *outputs.TLSConfig `config:"tls"`
+	TLS              *outputs.TLSConfig `config:"ssl"`
```

Comment:
I did consider renaming structs to SSL, but on the other hand, the module and all stdlib types do use TLS.